python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -5,23 +5,23 @@
 # $Id: LocalFile.py,v 0.1 2011-09-29 15:40:00 idzhunov Exp $
 ##########################################################################
 
+import errno
+import re
+import os
+from os import path
```

Comment:
Yes but it reduces the amount of something.somethingelse.something which gets annoying imo